buildpipeline/Core-Setup-CrossBuild.json

```diff
@@ -61,32 +61,34 @@
       "displayName": "Start detached container",
```

Comment:
Agreed. IIRC this is because this def was copied from CoreFX and tweaked for Core-Setup.